deepspeed/pt/deepspeed_cuda.py

```diff
@@ -393,21 +393,24 @@ def backward(ctx, grad_output):
 class DeepSpeedTransformerLayer(nn.Module):
     """Initialize the DeepSpeed Transformer Layer.
 
+        Static variable:
```

Comment:
The API doc (https://deepspeed.readthedocs.io/en/latest/kernel.html) will auto retrieve the strings here, if "layer_id" is invisible to users,  shouldn't add it here.

Reply:
good point. I will remove it then